### PR TITLE
Feature/buttons-keili

### PR DESCRIPTION
**Pull request by keikeka**
Branch to create additional buttons for the web.

Created logo button for home page. 
Created Favorite button.

### CODE REVIEW

File: src/js/component/Buttons/Favorite/Favorite.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+       alt="Image Favorite Button" 
+       height="52"
+       width="52"/>
+      <img 
+       className="favorite-golden"
+       src={GoldenFavorite} 
+       alt="Image Favorite Button Active" 
+       height="52"
+       width="52"/>
+    </div>
```
Comment (OvidioSantoro) [inline src/js/component/Buttons/Favorite/Favorite.jsx]: Only one of those will be rendered at any given time. Each one of these should be a different component: one for the empty star and another for the golden one. If we manage to use it as SVG we could use fill it with CSS, but for the moment, we need two different components.

Comment (OvidioSantoro) [reply src/js/component/Buttons/Favorite/Favorite.jsx]: Also, whenever is possible, let's try and avoid using inline style properties (e.g: `"height="52" width="auto"`). In this particular case, these buttons are embedded in the Coll, we have a global class for them, in order to control the size of all buttons easily: `className="button-coll"`

File: src/js/component/Buttons/UCollLogo/UCollLogo.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const UCollLogo = () => {
+  return (
+    <>
+    <div className="button-ucolllogo">
+       <img        
+       src={UCollLogoLogoVertical} 
+       alt="Image ucolllogo Button" 
+       height="100"
+       width="auto"/>
```
Comment (OvidioSantoro) [inline src/js/component/Buttons/UCollLogo/UCollLogo.jsx]: Again, let's try and use CSS for all styling.

**Review by OvidioSantoro — CHANGES_REQUESTED** [review]
I commented a pair of things about your PR, they're easy to solve!

**Review by OvidioSantoro — APPROVED** [review]

**Review by AslanSN — APPROVED** [review]
All seems fine, tested in my local computer all worked as it should!

**Pull request merged**